core/src/main/java/com/google/zxing/client/result/WifiParsedResult.java

```diff
@@ -27,17 +27,29 @@
   private final String networkEncryption;
   private final String password;
   private final boolean hidden;
+  private final String identity;
+  private final String anonymousIdentity;
+  private final String eapMethod;
+  private final String phase2Method;
```

Comment:
I was going to say we should probably break down this class rather than loading all these different properties into one place, but at this point, probably worth just tacking it on.